index.html

```diff
@@ -345,8 +345,35 @@ <h2>General rules</h2>
     </section>
 		<section id="roleMappingComputedRole">
 		  <h2>Computed Role</h2>
-                  <p>The computed role of an element is a string that represents the role of the element for the purposes of specification comformance testing. When an element has a role but is not contained in the required context (for example, a role `listitem` used on an element outside of a `list`), the computed role of the element is unspecified.</p>
-                  <p class="note">User agents can provide this role string, for example, in response to the WebDriver function <a href="https://w3c.github.io/webdriver/#get-computed-role">`getComputedRole`</a>.</p>
+      <p>The `computedrole` of an element is a string that represents the role of the element as computed by the browser engine. The `computedrole` is used primarily for the purposes of developer tools and specification comformance and interoperability testing.</p>
+      <p class="note">User agents provide this role string, for example, in developer tools, and in response to the WebDriver function <a href="https://w3c.github.io/webdriver/#get-computed-role">`getComputedRole`</a>, which is used for <a href="https://github.com/w3c/aria/blob/main/documentation/tests.md">interoperability testing of ARIA, HTML-AAM, and other specifications</a>.</p>
+      <aside class="example">
+
+<!-- ReSpec needs these examples to be unindented. -->
+<pre>&lt;button&gt; &lt;!-- computedrole returns "button" --&gt;
+
+&lt;a href="#" role="button"&gt; &lt;!-- computedrole returns "button" --&gt;</pre>
+
+      </aside>
+      <p>When an element has a role but is not contained in the required context (for example, an orphaned `listitem` used outside of a `list` context), User Agents MUST ignore the role token, and return the `computedrole` as if the ignored role token had not been included.</p>
```

Comment:
```suggestion
      <p>When an element has a role but is not contained in the required context (for example, an orphaned `listitem` without the required accessible parent of role `list`), User Agents MUST ignore the role token, and return the `computedrole` as if the ignored role token had not been included.</p>
```

Reply:
scott suggested this last time but I missed it, what we are talking about here is required accessibility parent, so I think we should say that explicitly. I'm not sure where it should link to, however -- I mean, I don't know how to link to a concept in the editors draft?

Reply:
agreed. marking as resolved.